docs/build/program/_index.md

```diff
@@ -38,12 +38,16 @@ Next, to install your preferred Viam SDK on your Linux or macOS development mach
 
 If you are using the Python SDK, [set up a virtual environment](/build/program/python-venv/) to package the SDK inside before running your code, avoiding conflicts with other projects or your system.
 
+For macOS (both Intel `x86_64` and Apple Silicon) or Linux (`x86`, `aarch64`, `armv6l`), run the commands outlined below.
```

Comment:
Vale should be underlining "below". Both below and above aren't great because it doesn't work for blind people. It's more accurate to say the following 
```suggestion
For macOS (both Intel `x86_64` and Apple Silicon) or Linux (`x86`, `aarch64`, `armv6l`), run the following commands:
```